I.write-tests/05-get-second-third-smallest/get-second-third.test.js

```diff
@@ -1,3 +1,15 @@
+var myNewArray = require('./get-second-third');
+test('Get secondandthird', function () {
+    const array = [90, 5, 11, 8, 6];
+    const expectation = [6, 8];
+    const result = myNewArray(array)
+    expect(result).toEqual(expectation);
```

Comment:
this test is not passing.
The error message says that the result was `[5, 11]`.
Try debugging the array that is being *sorterd* to see if the behaviour is what you expected.
You may want to double check the documentation of [Array.prototype.sort()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to understand why this is happening.

Reply:
I checked the code and is passing, i didn't received an error

Reply:
That is interesting. Maybe we are not running the same command and it could be that I am missing something :)

Could you please send me the command you are running for your tests?
Also, could you try making sure that you have pushed all your changes by running `git status` and confirmat that `everything is up to date`?
Finally, could you run `npm test` to make sure that you run all the tests and all of them pass?

Reply:
I did  nmp test and the result is below


coder@codeyourfuture:~/js-exercises-tdd$ npm test



> tdd@1.0.0 test /home/coder/js-exercises-tdd
> jest

 PASS  II.pass-tests/10-paint-ford/paint-cars.test.js
 PASS  II.pass-tests/09-car-sales/car-sales.test.js
 PASS  II.pass-tests/ES-6-practice/es6.test.js
 PASS  II.pass-tests/11-cities-formatter/cities.test.js
 PASS  II.pass-tests/05-factorial/factorial.test.js
 PASS  I.write-tests/09-test-async/async-2.test.js
 PASS  I.write-tests/02-remove-vowels/remove-vowels.test.js
 PASS  I.write-tests/01-greet-people/greet-people.test.js
 PASS  I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js
 PASS  II.pass-tests/06-remove-middle/remove-middle.test.js
 PASS  I.write-tests/04-numbers-greater-than-10/numbersGreaterThan10.test.js
 PASS  I.write-tests/06-largest-number/largest-number.test.js
 PASS  II.pass-tests/01-add-one/add-one.test.js
 PASS  II.pass-tests/07-second-largest/second-largest.test.js
 PASS  I.write-tests/07-get-even-numbers/get-even-numbers.test.js
 PASS  I.write-tests/05-get-second-third-smallest/get-second-third.test.js
 PASS  II.pass-tests/02-get-word-lengths/get-word-lengths.test.js
 PASS  II.pass-tests/04-find-needle/find-needle.test.js
 PASS  II.pass-tests/03-sum-numbers/add-numbers.test.js
 PASS  II.pass-tests/08-get-average/get-average.test.js
 PASS  III.tdd-katas/fizz-buzz/fizz-buzz.test.js
 PASS  I.write-tests/09-test-async/async-1.test.js

Test Suites: 22 passed, 22 total
Tests:       31 passed, 31 total
Snapshots:   0 total
Time:        15.615s
Ran all test suites.
coder@codeyourfuture:~/js-exercises-tdd$

Reply:
Thanks for your valuable comments

Reply:
I've done a few tests on my machine.
The default version of Node that I was running is **11.8.0**. After I downgraded it to **8.0.0** and even to **10.15.1**, the tests passed.
This is a really strange behaviour of Node, but certainly not an issue with your code.
You can run `node -v` to figure out what version of Node you are running.
Thanks for all the info :+1: